runner.go

```diff
@@ -178,19 +184,19 @@ func Run(options ...RunOption) {
 		return
 	}
 
-	log.Println("Using config", c.ConfigPath)
 	taskFlagGroups := runtime.groupTaskAndFlagArgs(os.Args[1:])
 	var desiredTasks []string
 	for taskName := range taskFlagGroups {
 		desiredTasks = append(desiredTasks, taskName)
 	}
 	var watchMode bool
 	if len(desiredTasks) == 0 {
-		desiredTasks = c.DesiredTasks
-		watchMode = !nonInteractive
+		log.Fatalln("Error: invalid target task(s)")
```

Comment:
Does running `taskrunner` break now?

Reply:
Yeah, good catch - I think it will. I'll follow up to confirm/fix this.